src/analysis/complex/basic.lean

```diff
@@ -189,3 +190,65 @@ by simp [is_R_or_C.norm_sq, complex.norm_sq]
 by simp [is_R_or_C.abs, complex.abs]
 
 end is_R_or_C
+
+section complex_fderiv_properties
+
+open complex continuous_linear_map
+
+variables {f : ℂ → ℂ} {z : ℂ}
+
+lemma fderiv_eq_fderiv_of_holomorph (h : differentiable_at ℂ f z) :
```

Comment:
I don't know the conventions... have we established to write `of_holomorph` for complex differentiable functions?